tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+	"compilerOptions": {
+		"allowJs": true,
+		"esModuleInterop": true,
+		"forceConsistentCasingInFileNames": true,
+		"importHelpers": true,
+		"module": "esnext",
+		"moduleResolution": "node",
+		"noImplicitReturns": true,
+		"removeComments": true,
+		"resolveJsonModule": true,
+		"strict": true,
+		"target": "es2020"
+	},
+	"include": [
+		"src/**/*",
+		"scripts/**/*"
+	]
+}
```

Comment:
I use the a simpler config across all my other packages, e.g. https://github.com/octokit/core.js/blob/master/tsconfig.json

Could we use the same one here?

Reply:
> I use the a simpler config across all my other packages, e.g. [octokit/core.js:tsconfig.json@`master`](https://github.com/octokit/core.js/blob/master/tsconfig.json) [](/octokit/core.js/blob/HEAD@{2020-04-28T23:19:47Z}/tsconfig.json)
> 
> Could we use the same one here?

Yep sure! The only one which was mandatory was the one for resolving JSON but, after restoring JavaScript on `/scripts/*` no necessary anymore.